source/louisHelper.py

```diff
@@ -1,12 +1,16 @@
-#louisHelper.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2018 NV Access Limited, Babbage B.V.
+# louisHelper.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2018-2021 NV Access Limited, Babbage B.V., Joseph Lee
 
 """Helper module to ease communication to and from liblouis."""
 
-import louis
+# Python 3.8 changes the way DLL's are loaded due to security.
+# Thus manually add NVDA executable path to DLL lookup path for loading liblouis.dll.
+import os
+with os.add_dll_directory(os.path.dirname(__file__)):
```

Comment:
`os.path.dirname(__file__)` is not gonna work for binary copies since it would point to nvda_install_dir\library.zip. Please use `globalVars.appDir` instead.